### DISCUSSION TITLE
Update Part doesn't work with me

### DISCUSSION THREAD

**Discussion by Toantd00** [BartoszCichecki/LenovoLegionToolkit#1475] [Q&A]
![{3DE2527B-C68E-4A05-BBBE-3C4BD7F2F38B}](https://github.com/user-attachments/assets/0d2d5bdb-bcd4-4118-9c96-ff426463d495)
I can't use update in Levono Toolkit, AMD Ryzen 7 8745H chip, Legion 5 2024. Internet connected normarly

**Comment by BartoszCichecki** [comment]
Continues is #1476
